src/components/GradeBookContentContainer.tsx

```diff
@@ -3,17 +3,17 @@ import styled from 'styled-components';
 import ActionButton from './ActionButton';
 import { Course, CourseItem } from '../api';
 import GradeItemAccordion from './GradeItemAccordion';
+import GradeBookHeader from './GradeBookHeader';
```

Comment:
This is causing the build error since it is never used (and warning are errors in our build pipeline)